bot/exts/utilities/challenges.py

```diff
@@ -0,0 +1,242 @@
+import logging
+from random import choice
+
+from bs4 import BeautifulSoup
+from discord import Color, Embed, Interaction, Message, SelectOption, ui
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, Emojis, NEGATIVE_REPLIES
+
+logger = logging.getLogger(__name__)
+API_ROOT = "https://www.codewars.com/api/v1/code-challenges/{kata_id}"
+
+
+class Challenges(commands.Cog):
+    """
+    Cog for a challenge command.
+
+    Pulls a random kata from codewars.com, and can be filtered through.
```

Comment:
I feel like there's room for improvement over this.

Reply:
How does this sound
```
"""
Cog for a challenge command.

The challenge command pulls a random kata from codewars.com. A kata is the name for a challenge, specific to `codewars.com`

The challenge command also has filters to customize the kata that is given. You can specify the language the kata should be from, the difficulty of the kata (1 is the hardest, 8 is the easiest, on a scale from 1-8), and lastly you can customize the topic you want the kata to be about
"""
```

Reply:
Sounds good to me.